src/main/java/org/threeten/extra/Interval.java

```diff
@@ -477,7 +477,7 @@ public Interval span(Interval other) {
     /**
      * Checks if this interval is after the specified instant.
      * <p>
-     * The result is true if the this instant starts after the specified instant.
+     * The result is true if the start of this instant starts after the specified instant.
```

Comment:
More probably should have been "The result is true if **~~the~~** this **interval** starts after the specified instant.".  etc for the other changes.

Reply:
Sounds good to me. Updated the changes to reflect what you've suggested.

Thank you!
Bruno